src/he/conn.c

```diff
@@ -487,7 +491,7 @@ he_return_code_t he_conn_send_keepalive(he_conn_t *conn) {
     return HE_ERR_NULL_POINTER;
   }
 
-  if(conn->state != HE_STATE_ONLINE) {
+  if(conn->state != HE_STATE_ONLINE || conn->pmtud_state != HE_PMTUD_STATE_DISABLED) {
```

Comment:
Actually, since we are using different variable for the keepalive & pmtud pending ids, there's no reason why we can't do keepalive during PMTUD, maybe we should just remove the `conn->pmtud_state != HE_PMTUD_STATE_DISABLED` check here?

Reply:
Done.